containers/src/Kokkos_Bitset.hpp

```diff
@@ -62,13 +63,11 @@ class Bitset {
       BIT_SCAN_REVERSE | MOVE_HINT_BACKWARD;
 
  private:
-  enum : unsigned {
-    block_size = static_cast<unsigned>(sizeof(unsigned) * CHAR_BIT)
-  };
-  enum : unsigned { block_mask = block_size - 1u };
-  enum : unsigned {
-    block_shift = Kokkos::Impl::integral_power_of_two(block_size)
-  };
+  static constexpr unsigned block_size = sizeof(unsigned) * CHAR_BIT;
+  static constexpr unsigned block_mask = block_size - 1u;
+  static constexpr unsigned block_shift =
+      Kokkos::has_single_bit(block_size) ? Kokkos::bit_width(block_size) - 1
```

Comment:
I'm assuming there is no warning under any compiler for a conditional operator combining `int` and `unsigned`.  Anyway, not a blocker.

Reply:
The CI will tell us.
I thought the standard says that it picks the common type.